Update autocomplete.html

ItWorksOnMyMachine opened the pull request:
In relation to issue #37.

ItWorksOnMyMachine committed:
Update autocomplete.html

ItWorksOnMyMachine commented:
the lastFindPromise is defined on the view model only after the first call to valueChanged, which doesn't happen until after the first setFocus. This causes the "No results" list item to show because the truthy type coercion of undefined results in false and !lastFindPromise ends up being true.

RWOverdijk commented on src/component/bootstrap/autocomplete.html:
```diff
@@ -27,7 +27,7 @@
 
       <li
-        show.bind="results.length === 0 && !lastFindPromise"
+        show.bind="results.length === 0 && lastFindPromise != undefined && !lastFindPromise"
```
Please use !==

RWOverdijk commented:
Cool nick!

One question, could you use our commit message format? https://github.com/SpoonX/about/blob/master/CONTRIBUTING.md#commit

ItWorksOnMyMachine committed:
Update autocomplete.html

fix(autocomplete): Update binding to rake into account initial undefined state

The lastFindPromise is defined on the view model only after the first call to valueChanged, which doesn't happen until after the first setFocus. This causes the "No results" list item to show because the truthy type coercion of undefined results in false and !lastFindPromise ends up being true.

This is in relation to Issue #37

ItWorksOnMyMachine commented:
Sorry about the comment, this is my first pull request and I totally missed the requirement.  I've updated the comment and changed the non-equality to a strict non-equality for consistency.
